### PR TITLE
Added commands for RHEL

### PR DESCRIPTION
**Pull request by imvpn22**
More general for every linux distro.

### CODE REVIEW

**Comment by texas2010** [comment]
I think that title is too long.  more like keep it simple.

**Comment by imvpn22** [comment]
Please suggest a simple title, edit if possible.

File: src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md [hunk 1]
```diff
@@ -1,15 +1,56 @@
 ---
-title: How to Create or Delete a User or Sudo User on Ubuntu
+title: How to Create/Delete a User or Sudo User on Linux
```
Comment (davcri) [inline src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md]: What about a simple `User management on Linux` ?

**Comment by texas2010** [comment]
if we change title. more like we have to change folder name too.

**Comment by davcri** [comment]
Asbolutely! @imvpn22 can you update the folder name and the article's title ?

File: src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md (hunk 1 above)
Comment (imvpn22) [reply src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md]: It's also in there.

Comment (imvpn22) [reply src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md]: That's good.

Comment (imvpn22) [reply src/pages/linux/how-to-create-a-sudo-user-on-ubuntu/index.md]: Fixed the title along with folder name

**Comment by imvpn22** [comment]
Please review the title and content now, and suggest me if anything I can change.

**Comment by davcri** [comment]
Sorry, I was trying to fix a minr issue (at the end of the file) and messed up the pull request:
I shouldn't have to update from `fcc/master` when pushing new committs to `imvpn22/patch-1`. I'll handle it.

**Comment by davcri** [comment]
@imvpn22 can you please make a new pull request from you [patch-1 branch](https://github.com/imvpn22/guides/tree/patch-1) ?  

The branches seem all right, changes (are preserved)[https://github.com/imvpn22/guides/blob/patch-1/src/pages/linux/user-management-on-linux/index.md]. I don't know why Github shows all these changes. 

Sorry again for the trouble.

**Comment by imvpn22** [comment]
Which branch I have to make a pull request for?
@davcri

**Comment by davcri** [comment]
From your `patch-1` to fcc's `master`

**Comment by imvpn22** [comment]
Made PR #3014

**Comment by imvpn22** [comment]
Can I also merge my `master` and `patch-1` ?

**Comment by davcri** [comment]
Your changes are in the guides, so you can do whatever you want with your fork 😄

**Pull request closed without merging**